sklearn/linear_model/_ridge.py

```diff
@@ -103,6 +104,8 @@ def _mv(x):
                 # old scipy
                 coefs[i], info = sp_linalg.cg(C, y_column, maxiter=max_iter,
                                               tol=tol)
+        if callbacks is not None:
```

Comment:
maybe you can remove this since the `is not None` check is also done in `_eval_callbacks`